qualtran/bloqs/multi_control_multi_target_pauli.py

```diff
@@ -137,6 +140,160 @@ def _has_unitary_(self) -> bool:
         return True
 
 
+@frozen
+class MultiControlX(Bloq):
+    r"""Implements multi-control, single-target X gate as a bloq using $n-1$ clean ancillas.
+
+    References:
+        [Constructing Large Controlled Nots]
+        (https://algassert.com/circuits/2015/06/05/Constructing-Large-Controlled-Nots.html)
+    """
+
+    cvs: Tuple[int, ...] = field(converter=lambda v: (v,) if isinstance(v, int) else tuple(v))
+
+    @cached_property
+    def signature(self) -> 'Signature':
+        if len(self.cvs) > 0:
+            return Signature([Register('ctrl', bitsize=len(self.cvs)), Register('x', bitsize=1)])
+        else:
+            return Signature([Register('x', bitsize=1)])
+
+    def on_classical_vals(
+        self, x: 'ClassicalValT', **vars: 'ClassicalValT'
+    ) -> Dict[str, 'ClassicalValT']:
+        if len(self.cvs) > 0:
+            ctrl = vars['ctrl']
+        else:
+            return {'x': (x + 1) % 2}
+
+        ctrls = bits_to_ints(self.cvs)
```

Comment:
Document the `cvs` attribute. They're supposed to be 0s and 1s, right? i.e. bits